GiniVision/Classes/SupportedFormatsViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  SupportedTypesViewController.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 10/19/17.
+//  Copyright © 2017 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+final class SupportedFormatsViewController: UITableViewController {
+    
+    let reuseIdentifier = "reuseIdentifier"
+    let rowHeight: CGFloat = 70
+    let sectionHeight: CGFloat = 70
+    var sections: [(title: String, items: [String], itemsImage:UIImage?, itemsImageBackgroundColor: UIColor)] = [
```

Comment:
I think this is getting a little too complicated for tuples. Have you considered making a small struct for it?

Also, the `sections` is configurable by the library's users, right? Meaning, they can change it?

Reply:
No, the sections are not customizable bu the clients, only the color. 
I think that a `typealias` would be enough for this case.